src/internal/connector/discovery/api/sites.go

```diff
@@ -116,23 +178,23 @@ func validateSite(item any) (models.Siteable, error) {
 		return nil, clues.New("missing ID")
 	}
 
-	url := ptr.Val(m.GetWebUrl())
-	if len(url) == 0 {
+	wURL := ptr.Val(m.GetWebUrl())
+	if len(wURL) == 0 {
 		return nil, clues.New("missing webURL").With("site_id", id) // TODO: pii
 	}
 
 	// personal (ie: oneDrive) sites have to be filtered out server-side.
-	if strings.Contains(url, personalSitePath) {
+	if strings.Contains(wURL, personalSitePath) {
 		return nil, clues.Stack(errKnownSkippableCase).
-			With("site_id", id, "site_url", url) // TODO: pii
+			With("site_id", id, "site_web_url", wURL) // TODO: pii
```

Comment:
nit: this message and the one below are inconsistent for the web URL key (site_web_url vs site_url)